ReactWindows/ReactNative.ChakraBridge/ChakraStringResult.h

```diff
@@ -0,0 +1,13 @@
+#pragma once
+
+using namespace Platform;
+
+namespace ReactNative { namespace ChakraBridge {
```

Comment:
@rozele Nope, I just don't like multiple indents just for namespaces so I kept it on one line